backend/src/database/repositories/organizationRepository.ts

```diff
@@ -59,6 +59,22 @@ class OrganizationRepository {
     return this.findById(record.id, options)
   }
 
+  static async bulkUpdate<T extends any[]>(
+    data: T,
+    fields: string[],
+    options: IRepositoryOptions,
+  ): Promise<T> {
+    const isValid = new Set(data.filter((org) => org.id).map((org) => org.id)).size !== data.length
+    if (isValid) return [] as T
+
+    const orgs = await options.database.organization.bulkCreate(data, {
```

Comment:
I might be misunderstanding something, but the function name is `bulkUpdate` and the operation here is `bulkCreate`. Is this correct? If so, please add a comment explaining why 🙂

Reply:
`bulkCreate` can be used to create/update bulk. It updates on conflict with primary key or unique field when the `updateOnDuplicate` args is used. Comments have been added to the code to reflect this behavior.